outlook_calendar_service/app_config.py

```diff
@@ -0,0 +1,15 @@
+import os
+
```

Comment:
https://pydantic-docs.helpmanual.io/usage/settings/
consider this one for configuration ;)